pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -133,6 +140,15 @@ stdenv.mkDerivation rec {
     sha256 = "1qjr02cqncv20abdp1yzr55n7smhx6h9chqvb0xbp18byynvj87w";
   };
 
+  # remove this when updating to 0.3.16
+  patches = [
+    (fetchpatch {
+      name = "riscv64-fixes.patch";
+      url = "https://github.com/xianyi/OpenBLAS/pull/3259.patch";
```

Comment:
Can we fetch by referencing the exact commit ids instead? Pull requests are tracking a branch, that could, in theory, still move.